cmd/soroban-rpc/internal/methods/get_transaction.go

```diff
@@ -88,16 +89,22 @@ func GetTransaction(getter transactionGetter, request GetTransactionRequest) (Ge
 		}
 	}
 
-	tx, found, storeRange := getter.GetTransaction(txHash)
+	tx, storeRange, err := reader.GetTransaction(ctx, txHash)
+
 	response := GetTransactionResponse{
 		LatestLedger:          storeRange.LastLedger.Sequence,
 		LatestLedgerCloseTime: storeRange.LastLedger.CloseTime,
 		OldestLedger:          storeRange.FirstLedger.Sequence,
 		OldestLedgerCloseTime: storeRange.FirstLedger.CloseTime,
 	}
-	if !found {
+	if err == db.ErrNoTransaction {
```

Comment:
better to use https://pkg.go.dev/errors#Is in case the error is wrapped

Reply:
great catch! done in 9fc64a9